CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 Release Notes
 -------------
 
+1.9.0 (2020-05-01)
```

Comment:
Replace the date with `unreleased`.

However (and whenever) does the release will set the date. 😊